tests/testthat/_snaps/board_rsconnect_bundle.md

```diff
@@ -34,6 +34,23 @@
           </style>
         </head>
         <body>
+      
+          <section>
+             <h3>TEST/test</h3>
+             <p>
+               Last updated:  •
```

Comment:
Would be good to tweak the test so that we get semi-realistic data here

Reply:
Agreed - I've updated the test accordingly